README.md

```diff
@@ -229,6 +236,13 @@ We also recommend using sex assignments generated from the ploidy estimates and
 * Ploidy estimates, sex assignments, with plots
 * (Optional) Outlier samples detected by call counts
 
+#### <a name="prelim-sample-qc">Preliminary Sample QC</a>
+The purpose of sample filtering at this stage after Module00b is to prevent very poor quality samples from interfering with the results for the rest of the callset. In general, samples that are borderline are okay to leave in, but you should choose filtering thresholds to suit the needs of your cohort and study. There will be future opportunities (as part of [Module03](#module03)) for filtering before the joint genotyping stage if necessary. Here are a few of the basic QC checks that we recommend:
+* Look at the X and Y ploidy plots, and check that sex assignments match your expectations. If there are discrepancies, check for sample swaps and update your PED file before proceeding.
+* Look at the dosage score (WGD) distribution and check that it is centered around 1. Optionally filter outliers.
```

Comment:
Does this advice apply if your cohort contains both PCR+ and - samples? (although I bet most people won't have a mix)

Reply:
Whoops I meant zero not one! Fixed in the next commit. It does apply though - the aggregated samples in a mixed cohort should be centered around 0, but we actually expect the PCR- distribution to be a little lower and the PCR+ distribution to be a little higher. I'll add a note to this effect.